lib/cisco_node_utils/evpn_vni.rb

```diff
@@ -37,6 +37,7 @@ def initialize(vni, instantiate=true)
     # Creat a hash of all vni instance
     def self.vnis
       hash = {}
+      RouterBgp.enable
```

Comment:
This is wrong. You should not be enabling a feature as part of a get-all-instances method. If the feature is disabled then the method just returns an empty hash.